bindings/wasm/native/Cargo.toml

```diff
@@ -1,16 +1,12 @@
-[workspace]
-
 [package]
-name = "client-wasm"
+name = "iota-client-wasm"
```

Comment:
I think here it's also related to the npm package name, but needs to be verified